1.train_models/log_reg_plates_3_3p_5_cp_norm_data/nbconverted/plate_3_3p_5_train_log_reg_scv.py

```diff
@@ -291,12 +288,18 @@ def process_plates(_df):
         preds = logreg.predict(X_val)
         acc += accuracy_score(y_val, preds)
 
+        # Store model data for folds
+        eval_data["plate"].extend(restdf.iloc[val_index]["Metadata_Plate"].tolist())
+        eval_data["datasplit"].extend(["val"] * val_index.shape[0])
```

Comment:
Consider the use of [`np.repeat()`](https://numpy.org/doc/stable/reference/generated/numpy.repeat.html) here.

Reply:
This could be beneficial for decreasing runtime, I may have to look into this along with the above data translation suggestion.